src/FilamentManager.php

```diff
@@ -193,4 +195,14 @@ public function version()
 
         return InstalledVersions::getPrettyVersion('filament/filament');
     }
+
+    public static function ignoreMigrations()
+    {
+        static::$shouldRunMigrations = false;
+    }
+
+    public static function runsMigrations()
```

Comment:
```suggestion
    public function shouldRunMigrations()
```

This is a naming convention used elsewhere in Filament so we should keep it consistent.